addons/OPT/GPS/fn_clientInit.sqf

```diff
@@ -0,0 +1,120 @@
+#include "macros.hpp"
+
+// Inspired by AAW's UnitTracker
+
+GVAR(playerCounter) = 0;
+GVAR(currentIcons) = [];
+GVAR(blockUpdate) = false;
+GVAR(currentHoverGroup) = grpNull;
+GVAR(currentHoverVehicle) = objNull;
+GVAR(groupInfoPFH) = -1;
+GVAR(vehicleInfoPFH) = -1;
+GVAR(lastFrameTriggered) = diag_frameNo;
+
+GVAR(processedIcons) = [];
+GVAR(lastProcessedIcons) = [];
+
+
+// Register Beam dialog map to show unit markers too. Only try doing this once the mission actually started, so everything is hopefully in place.
+// Because we can only do this for an existing display/control, we gotta wait until it actually exists
+["missionStarted", {
+    [{
+        ((findDisplay 444001) displayCtrl 10007) call CFUNC(registerMapControl);
```

Comment:
die IDD/IDC sind momentan hardcoded in der Mission. In Zukunft würde es fürs Registrieren des Trackers/GPS eine Extra-Funktion geben, die genau das macht.